content/cpp/concepts/references/references.md

```diff
@@ -1,20 +1,20 @@
 ---
-Title: 'References'
-Description: 'A reference variable is an alias for another object. It is created using the & sign. Two things to note: Anything done to the reference also happens to the original and aliases cannot be changed to alias something else.'
+Title: "References"
+Description: "A reference variable is an alias for another object. It is created using the & sign. Two things to note: Anything done to the reference also happens to the original and aliases cannot be changed to alias something else."
```

Comment:
Keep the description less than or equal to 160 characters. Avoid using colon (:). Also, the description should be inside single quotes('').